src/machine-learning/machine-learning.ts

```diff
@@ -252,13 +295,73 @@ export class ModelOptions {
   public displayName?: string;
   public tags?: string[];
 
-  public tfliteModel?: { gcsTFLiteUri: string; };
+  public tfliteModel?: { gcsTfliteUri: string; };
+}
 
-  protected toJSON(forUpload?: boolean): object {
-    throw new Error('NotImplemented');
+async function convertOptionstoContent(
```

Comment:
Make this part of the class so it can access `appInternal`.

Reply:
done